pkg/ddc/alluxio/transform_resources_test.go

```diff
@@ -26,6 +27,89 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/log"
 )
 
+func TestTransformResourcesForMaster(t *testing.T) {
+	testCases := map[string]struct {
+		runtime *datav1alpha1.AlluxioRuntime
+		got     *Alluxio
+		want    *Alluxio
+	}{
+		"test alluxio master pass through resources with limits and request case 1": {
```

Comment:
Did you test the case without passing resourceRequirements?

Reply:
ok, fill test case where no resource is passed